tmt/steps/provision/testcloud.py

```diff
@@ -1073,11 +1057,18 @@ def start(self) -> None:
         _apply_hw_disk_size(self.hardware, self._domain, self._logger)
         _apply_hw_tpm(self.hardware, self._domain, self._logger)
 
-        self.debug('final domain memory', str(self._domain.memory_size))
-        self.debug('final domain root disk size', str(self._domain.storage_devices[0].size))
+        self.info(
+            'memory', f'{tmt.hardware.UNITS(f"{self._domain.memory_size} kB").to("MB")}', 'green'
+        )
+        self.info(
+            'disk',
+            f'{tmt.hardware.UNITS(f"{self._domain.storage_devices[0].size} GB").to("GB")}',
+            'green',
+        )
 
-        for i, device in enumerate(self._domain.storage_devices):
-            self.debug(f'final domain disk #{i} size', str(device.size))
+        for i, device in enumerate(self._domain.storage_devices, start=1):
+            size_gb = tmt.hardware.UNITS(f"{device.size} GB").to("GB")
+            self.debug(f'Domain disk {i} size: {size_gb}')
```

Comment:
`debug()` can also log key/value type of value, like `info()` does above, therefore `self.debug(f'Domain disk {i} size', tmt.hardware.UNITS(f"{device.size} GB").to("GB"))`. And you can save one extra variable.